### ISSUE TITLE
Need to add twitter(x) logo to navbar documentation next to github logo

### ISSUE THREAD

**Issue by antonmak1** [hmpl-lang/hmpl#30]
Here next to it you need to add svg with styles like those on GitHub. Logo - new. Link - https://x.com/hmpljs:
![Screenshot 2024-10-27 at 02 36 23](https://github.com/user-attachments/assets/d0cfd057-b31b-4a9b-9c82-0a502181c848)
An example:
![Screenshot 2024-10-27 at 02 34 30](https://github.com/user-attachments/assets/958337b6-631c-4c48-8dd1-6f95168f175c)

**Comment by NandaniThakur** [comment]
i want to work on this

**Comment by antonmak1** [comment]
@NandaniThakur Thanks a lot!

**Comment by antonmak1** [comment]
@NandaniThakur Hello! Please tell me, do you have time to do the task or should I reassign it?

**Comment by NandaniThakur** [comment]
Sorry, because of some personal issues I couldn't complete it. 
It'll be best if you can reassign it. 
Sorry for the inconvenience caused.

**Comment by antonmak1** [comment]
@NandaniThakur Yes, it's okay, it happens. I hope you solve your problems and everything will be fine with you!
